pkg/apis/operator/v1alpha1/validation/validation.go

```diff
@@ -96,6 +96,23 @@ func validateRuntimeClusterUpdate(oldGarden, newGarden *operatorv1alpha1.Garden)
 		allErrs = append(allErrs, apivalidation.ValidateImmutableField(oldRuntimeCluster.Ingress.Domains[0].Name, newRuntimeCluster.Ingress.Domains[0].Name, fldPath.Child("ingress", "domains").Index(0))...)
 	}
 
+	type network struct {
+		new, old []string
+		name     string
+	}
+	for _, n := range []network{
```

Comment:
You could make this anonymous:

```suggestion
	for _, network := range []struct {
		new, old []string
		name     string
	} {
```

Reply:
Addressed with https://github.com/gardener/gardener/pull/11251/commits/b6deee9ebc11d57d3fa77ff3adce91a6f0bd682a.